assets/src/infrastructure/ui/inputs/dateTime/Datepicker/style.scss

```diff
@@ -1,32 +1,216 @@
+					margin: 0 var(--ee-margin-micro);
+					padding: 0;
+				}
+
+				// the following style rules do the same thing, but need to be kept separate, else they no workie
+				input[type='number']::selection {
+					background-color: var(--ee-color-primary-low-contrast);
+					color: var(--ee-text-on-primary);
+				}
+				input[type='number']::-moz-selection {
+					background-color: var(--ee-color-primary-low-contrast);
+					color: var(--ee-text-on-primary);
+				}
+
+				&:active {
+					@extend .ee-date-picker-active;
+				}
+
+				&:focus {
+					@extend .ee-date-picker-focus;
+				}
+
+				&:hover {
+					@extend .ee-date-picker-hover;
+				}
+			}
+
+			&__wrapper {
+				border: none !important;
+
+				button {
+					background: var(--ee-button-background);
+					border-color: var(--ee-button-background);
+					border-radius: var(--ee-border-radius-small);
+					border-style: solid;
+					border-width: 3px;
+					box-shadow: none;
+					color: var(--ee-button-text-color);
+					height: var(--ee-icon-button-size);
+					margin: 0 var(--ee-margin-nano);
+					padding: var(--ee-padding-micro);
+					outline: none;
+					width: var(--ee-icon-button-size);
+
+					&:active {
+						@extend .ee-date-picker-active;
+					}
+
+					&:focus {
+						@extend .ee-date-picker-focus;
+					}
+
+					&:hover {
+						@extend .ee-date-picker-hover;
+					}
+					svg {
+						margin: 0;
+						path {
+							fill: currentColor;
+						}
+					}
+				}
+			}
+
+			&__calendar {
+				z-index: 10;
+
+				.react-calendar {
+					background: var(--ee-background-color);
+					border: 5px solid var(--ee-background-color);
+					box-shadow: var(--ee-box-shadow-big);
+					font-family: Montserrat, 'Segoe UI', Tahoma, sans-serif;
+					font-size: var(--ee-font-size-default);
+
+					&__navigation {
+						height: var(--ee-icon-button-size);
+						line-height: var(--ee-icon-button-size);
+
+						&__arrow {
+							background: var(--ee-background-color);
+							box-sizing: border-box;
+							color: var(--ee-button-text-color);
+							font-size: var(--ee-font-size-bigger);
+							height: var(--ee-icon-button-size);
+							line-height: 18px;
+							padding: 9px 5px 15px;
+							width: var(--ee-icon-button-size);
+
+							&:hover {
+								background: var(--ee-button-background);
+								color: var(--ee-button-text-color);
+							}
+
+							&:focus {
+								background: var(--ee-button-background);
+								color: var(--ee-default-text-color-high-contrast);
+							}
+						}
+
+						&__label {
+							font-weight: 700;
+						}
+					}
+
+					abbr[title] {
+						text-decoration: none;
+					}
+
+					&__month-view {
+						&__days {
+							&__day {
+								height: var(--ee-icon-button-size);
+								margin: 0;
+								width: var(--ee-icon-button-size);
+
+								&--neighboringMonth {
+									color: var(--ee-default-text-color-low-contrast);
+								}
+
+								&--weekend {
+									color: var(--ee-color-primary-low-contrast);
+								}
```

Comment:
nice that they followed BEM

Reply:
kinda sorta... they use multiple elements in the class names though, like:

```
react-calendar__month-view__days__day
```

so might be more accurate to say they use BEEEEEM 😆